CHANGES.md

```diff
@@ -1,6 +1,427 @@
 Release Changes List
 ====================
 
+o spellcheck
+o quotes check
+o URL docs check (1.26)
+o linebreak check
+o check compiler flags / man pages
+o check example codes
+
+version 1.26.0
+==============
+
+Twenty-ninth public release of Chapel, March 31, 2022
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* updated the Chapel compiler to support LLVM versions 11, 12, and 13
+* added a prototype ability to resize arrays of elements without default values
+* made stabilizing improvements to methods on ranges, domains, and arrays
+* added new 'CopyAggregation', 'Socket', and 'Channel' package modules
+* made stabilizing improvements to most standard library module interfaces
+* improved the portability, robustness, and performance of 'ofi' communication
+* expanded the generality of Chapel's ability to generate code for GPUs
+* added a new script for displaying available runtime configurations
+* many other bug fixes and improvements to error messages and documentation
+
+Packaging / Configuration Changes
+---------------------------------
+* the Chapel compiler's LLVM back-end now supports versions 11, 12, and 13
+* when multiple supported LLVM versions are found, the latest one is now chosen
+* updated error message when unable to find LLVM to list the supported versions
+* UTF-8 support no longer depends on the `LANG` environment variable  
+  (see https://chapel-lang.org/docs/1.26/usingchapel/chplenv.html#character-set)
+* removed support for the deprecated `CHPL_REGEXP` environment variable
+
+Syntactic / Naming Changes
+--------------------------
+* extended the set of statement types to which `try`/`try!` can be applied
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* modules from files listed on the command-line are now always initialized  
```

Comment:
Should "from" be "in" here?  "from" could indicate files that are imported or used in those files.

Reply:
Good point.  I've found this one of the most challenging items to fit to 80 characters because of all the qualifiers that apply (e.g., it's also only top-level modules, which command-line are we talking about, etc.).  How about:

```
* top-level modules defined in files on `chpl`'s command-line are now inited
```